src/kernel/include/shared/cos_config.h

```diff
@@ -119,4 +119,11 @@
 /* print out to dmesg? */
 /* #define COS_PRINT_DMESG 1 */
 
+/**
+ * Configuration to enable/disable functionality in Kernel.
+ */
+#define ENABLE_VGA
+#define ENABLE_SERIAL
+#define ENABLE_TIMER
```

Comment:
This commit conflicts with Jakobs.  Fix the merge conflict (sorry!).  I believe he removed `ENABLE_TIMER`.